src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Tables/DeltaTable.cs

```diff
@@ -220,6 +256,16 @@ internal DeltaTable(JvmObjectReference jvmObject)
         public DataFrame History() =>
             new DataFrame((JvmObjectReference)_jvmObject.Invoke("history"));
 
+        /// <summary>
+        /// Generate a manifest for the given Delta Table.
+        /// </summary>
+        /// <param name="mode">Specifies the mode for the generation of the manifest.
```

Comment:
I'm not sure how well Visual Studio will handle this multi-line `param` comment.  How does it look in IntelliSense?

Reply:
It looks well in IntelliSense. I have checked.